src/main/java/core/basesyntax/strategy/impl/BalanceServiceStrategyImpl.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.strategy.CountStrategy;
+
+public class BalanceServiceStrategyImpl implements CountStrategy {
```

Comment:
```suggestion
public class BalanceCountStrategyImpl implements CountStrategy {
```
same for other there impl

Reply:
fixed